src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', resolve);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(reject, 3000);
+});
+
+promise1.then(() => {
+  const div = document.createElement('div');
+
+  div.setAttribute('class', 'message');
+  div.innerText = 'Promise was resolved!';
+  document.body.appendChild(div);
+});
+
+promise1.catch(() => {
+  const div = document.createElement('div');
+
+  div.setAttribute('class', 'message error-message');
+  div.innerText = 'Promise was rejected!';
+  document.body.appendChild(div);
+});
```

Comment:
The catch block for promise1 will never be executed because promise1 only resolves on a click event and does not have a rejection path. If you intend to handle a rejection scenario, you need to implement a mechanism that can reject the promise.